bc/users/templates/register/login.html

```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
+
+{% block title %}Sign In{% endblock %}
+
+{% block content %}
+<div class="max-w-xl mx-auto px-4 sm:px-6 md:px-10">
+  <div class="py-12 prose">
+    <h1 class="font-display font-light tracking-wide">Sign In</h2>
+    {% if form.errors %}
+        {%  for error in form.non_field_errors %}
+        <p class="alert alert-danger">{{ error|safe }}</p>
+        {% endfor %}
+    {% endif %}
+
+    <form action="{% url "sign-in" %}" method="post">
+        {% csrf_token %}
+    <div class="grid grid-cols-1 gap-4">
```

Comment:
Indentation is a bit weird on this div. Also, we should be doing two spaces, not four in HTML, according to our editorconfig.